ssz/sedes/serializable.py

```diff
@@ -178,7 +183,22 @@ def copy(self, *args, **kwargs):
 
         combined_kwargs = dict(**unchanged_kwargs, **kwargs)
         all_kwargs = merge_args_to_kwargs(args, combined_kwargs, self._meta.field_names)
-        return type(self)(**all_kwargs)
+
+        result = type(self)(**all_kwargs)
+
+        if clear_cache:
+            result.clear_cache()
+
+        return result
+
+    def clear_cache(self):
+        if hasattr(self.db, 'reset_cache'):
+            self.db.reset_cache()
```

Comment:
`collections.abc.MutableMapping` defines `clear` as the method that does this.  Probably worth using that API.